app/services/samples/metadata/update_service.rb

```diff
@@ -97,6 +99,22 @@ def fields_not_updated(metadata_fields_update_status)
                      sample_name: @sample.name,
                      metadata_fields: metadata_fields_not_updated.join(', '))
       end
+
+      def update_metadata_summary
+        return unless @sample.previous_changes['metadata']
+
+        old_metadata = @sample.previous_changes['metadata'][0]
+        new_metadata = @sample.previous_changes['metadata'][1]
+
+        # Checks which keys are overlapping after metadata changes.
+        # Keys that are overlapping will not affect summary counts, therefore can be deleted from both hashes
+        old_metadata.each do |metadata_field, _v|
+          if new_metadata.key?(metadata_field)
```

Comment:
This should only happen if only values were changed correct? Otherwise there should be no overlap? If so the comment above to only run this method if an `addition` or `deletion` are made, then this loop shouldn't be required anymore

Reply:
This loop has now been removed in 7d1d6d693cca394dbacd24cdd02a4e1a8593c0e7
See comment about service overhaul.